src/global.store.ts

```diff
@@ -30,6 +31,7 @@ export class GlobalStore implements IGlobalStore {
         this._eagerPartnerStoreSubscribers = {};
         this._eagerUnsubscribers = {};
         this._actionLogger = new ActionLogger(_logger);
+        this._exposedDerivedState = {};
```

Comment:
Rename suggestion
`_exposedDerivedState` --> `selectors`

Reply:
Renamed, i added the `_` at the front to keep it consistent with the rest of the properties in this class, let me know what you think :)